### DISCUSSION TITLE
Event filter problem by page

### DISCUSSION THREAD

**Discussion by skymarocchinia** [plausible/analytics#1517] [Anything Else]
Good morning
I would need to retrieve through Stats API the data related to a specific event filtered by page
I tried so,
https://plausible.io/api/v1/stats/breakdown?site_id=xxx.net&period=7d&property=event:props:roomName&filters=event:page==/house/60f9799dc0b886002fa086e3
but it never returns results.
Am I doing something wrong?
Thank you
Alessandro

**Comment by RobertJoonas** [comment]
Hey! Is `roomName` a custom event property? In that case, if you would like a breakdown on this property, you should also define the name of this custom event in the filters. Something like this could work:

https://plausible.io/api/v1/stats/breakdown?site_id=xxx.net&period=7d&property=event:props:roomName&filters=event:name==nameOfYourCustomEvent;event:page==/house/60f9799dc0b886002fa086e3

Hope this helps!
